scripts/include.sh/build-dep.sh

```diff
@@ -155,14 +155,15 @@ build_git_ios()
 
 build_git_osx()
 {
-  sdk="`xcodebuild -showsdks 2>/dev/null | grep macosx | sed 's/.*macosx\(.*\)/\1/'`"
-  archs="x86_64"
+xc
```

Comment:
I am not sure what this `xc` command does here.

Reply:
🤷 me neither tbh